i18n/fr/faq.md

```diff
@@ -1,6 +1,32 @@
 # Foire Aux Qestions
 
-## Qu'est-ce que Kovri?
+## Table des matières
+- Questions générales
+    - Qu'est-ce que Kovri ?
+    - Qui développe Kovri ?
+    - En quoi Kovri va-t-il aider Monero ?
+    - Pourquoi devrais-je utiliser Kovri plutôt que I2P ?
+    - Qu'est-ce que Kovri permet ?
+    - Qu'est-ce qui est en cours de développement ?
+    - Qu'est-ce que Kovri ne permettra pas ?
+    - Quel est l'état actuel de développement de Kovri ?
+    - Quand sera disponible la version alpha ?
+    - Sur quoi se focalise en ce moment l'équipe de développement ?
+    - Kovri est-il déjà utilisable et quel niveau de vie privée propose-t-il actuellement ?
+    - Comment puis-je contacter les développeurs de Kovri ?
+- Kovri vs i2pd
+    - Pourquoi devrais-je utiliser Kovri plutôt que i2pd ?
+    - Quelles sont les principales différences entre Kovri et i2pd ?
+    - Pourquoi avez-vous forké à partir d'i2pd ?
+    - Quels sont les principaux facteurs qui ont amené à forker i2pd ? (Et pourquoi y a-t-il deux repos i2pd, un sur Bitbucker et un sur GitHub ?)
+- Utiliser Kovri
+    - J'ai trouvé une vulnérabilité, j'ai trouvé un bug, que dois-je faire ?
+    - Pourquoi mes logs montrent une heure différente de mon fuseau horaire ?
```

Comment:
Why all the spaces between the question marks?

Reply:
Because in french we have a space before the question mark ;-)

Reply:
Of all the languages I'm familiar with or have used over the years, French has always been dead-last (as in, almost never). Even when I did work with French-language materials, I had always read English-produced transliterations. On top of that, I'm more familiar with French-Canadian than indigenous France - so that doesn't help much 😆 

I should've known better and done a simple search 🤣 Thank you for clarifying, @woodyjon.